cg/apps/housekeeper/hk.py

```diff
@@ -390,6 +394,24 @@ def is_fastq_or_spring_in_all_bundles(self, bundle_names: List[str]) -> bool:
             )
         return all(sequencing_files_in_hk.values())
 
+    def get_non_archived_files(self, bundle_name: str, tags: Optional[list] = None) -> List[File]:
+        """Returns all files from given bundle, with given tag, which have not been archived."""
+        return self._store.get_non_archived_files(bundle_name=bundle_name, tags=tags)
+
+    def get_archived_files(self, bundle_name: str, tags: Optional[list] = None) -> List[Path]:
+        """Returns all files from given bundle, with given tag, which have been archived."""
+        return self._store.get_archived_files(bundle_name=bundle_name, tags=tags)
+
+    def add_archives(self, files: List[Path], archive_task_id: int) -> None:
+        """Creates an archive object for the given files, and adds the archive task id to them."""
+        for file in files:
+            archived_file: File = self._store.get_files(file_path=file.as_posix()).first()
```

Comment:
What if no file is found? Maybe we need to handle that case here

Reply:
Good idea 👍